Integration/cptIMSCobraExport_frm.frm

```diff
@@ -492,7 +490,7 @@ Private Sub CancelBtn_Click()
 End Sub
 
 Private Sub cptLinkLabel_Click()
-    If cptInternetIsConnected Then Application.FollowHyperlink "http://www.ClearPlanConsulting.com"
+    Application.OpenBrowser "http://www.ClearPlanConsulting.com"

```

Comment:
The above change from `OpenBrowser` to `FollowHyperlink` was implemented earlier -- not sure why it didn't make it through, but I've replaced it. The `OpenBrowser` method wasn't working.